src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -1354,14 +1354,14 @@ public void CreateAndCloneOrgCodeLists(string org)
         }
 
         /// <summary>
-        /// create a repository in gitea for the given organisation and options
+        /// create a repository in gitea for the given org and options
```

Comment:
and here

Reply:
Have used org since that variable is explained to more detail in the parameter list.